### DISCUSSION TITLE
Concurrent mode?

### DISCUSSION THREAD

**Discussion by howroyd** [howroyd/simonsays#44] [Ideas]
Toggleable concurrent mode so commands can execute in parallel.  Mainly to prevent commands queuing up and being pre-empted by the player by reading chat.

Probably shouldnt be default though so as a selectable option could be good.

**Comment by MorbidMorke** [comment]
I had an issue using it where things were being queued up via chat. The chaos of it all meant I couldn't keep up with the list, so I didn't know what was coming next specifically (plus the cooldowns I set on various things). I feel like a concurrent mode would be absolute, immeasurable chaos xD
